BankAccount/pkg/store/transactionhistory.go

```diff
@@ -1 +1,73 @@
 package store
+
+import (
+	"bankacc/pkg/entities"
+	"database/sql"
+	"time"
+)
+
+type TransactionHistoryStore interface {
+	Insert(userId int, accountId int, amount float64, action string) (*entities.TransactionHistory, error)
+	GetTransactionsById(id int) (*[]entities.TransactionHistory, error)
+	GetTransactionsByIdFromToDate(id int, fromDate time.Time, toDate time.Time) (*[]entities.TransactionHistory, error)
+}
+
+type TransactionHistoryModel struct {
+	Db *sql.DB
+}
+
+func NewTransactionHistoryStoreModel(db *sql.DB) *TransactionHistoryModel {
+	return &TransactionHistoryModel{
+		Db: db,
+	}
+}
+
+func (store *TransactionHistoryModel) Insert(userId int, accountId int, amount float64, action string) (*entities.TransactionHistory, error) {
+	transaction := entities.TransactionHistory{
+		UserId:    userId,
+		AccountId: accountId,
+		Amount:    amount,
+		Action:    action,
+		CreatedAt: time.Now(),
```

Comment:
@vpatcev18 do not use time.Now() twice, this can lead to different times, keep it in a variable like this:

```
now := time.Now()
```

and use this variable both times.

Reply:
@FilipNikolovski resolved